### ISSUE TITLE
new FAQ questions till we re:work FAQ system

### ISSUE THREAD

**Issue by yangwao** [kodadot/nft-gallery#192]
Here we can add FAQ questions that would be worthy to have on-site, till we will rework stuff.

**Comment by anettrolikova** [comment]
need more ELI5 - explain like I'm 5 explanations. 

Here is ELI5 on Collections : 
Collection is like photoalbum of artworks, If you made more artworks in one theme its better and cheaper to upload them as collectiion as 1- they will show as one collection together like photoalbum and 2- it wll be cheaper to mint them. Or you can even understand collection like groups and then think about them like lets say collection of sweet fruit where you put bananas, peaches, mangoes... then collection of sour fruit where you put oranges, lemons... or even colours - lets say collection of black cars, red cars and put all types of cars that you own in the same colour into one collection. 

ELI5 on Single NFT: 
single nft means that this art doesn't fits into any collection (photoalbum if that is more understandable) and you want to have it as alone.

**Comment by yangwao** [comment]
📟  @cryptodamsky
